config/taxonomy/objects/application.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "application.values.schema.json",
+    "title": "Application Values Taxonomy",
+    "type": "object",
+    "description": "Values contributed to by Fybrik applications.",
+    "properties": {
+        "spec": {
+            "type": "object",
+            "properties": {
+                "appInfo": {"$ref": "taxonomy.json#/definitions/AppInfo"},
```

Comment:
To make this work without the layers you need AppInfo in the base.yaml
When you generate the taxonomy.json with layers it will add checks for Intent and Role.

Reply:
Yes this application.json is assuming that taxonomy.json is compiled with the AppInfo layer

Reply:
That is an issue for our out-of-the-box deployment that uses only the base yaml.